astropy/timeseries/periodograms/lombscargle/implementations/scipy_impl.py

```diff
@@ -57,7 +61,12 @@ def lombscargle_scipy(t, y, frequency, normalization="standard", center_data=Tru
         y = y - y.mean()
 
     # Note: scipy input accepts angular frequencies
-    p = signal.lombscargle(t, y, 2 * np.pi * frequency)
+    # floating_mean=True will fail tests due to https://github.com/scipy/scipy/pull/21277
```

Comment:
this comment might become confusing and unnecessary if `floating_mean=True` never becomes the default, as is the direction it's taking upstream. We'll probably want to amend or simply remove it.